lib/proxy_api/proxy_exception.rb

```diff
@@ -8,7 +8,15 @@ def initialize(url, exception, message, *params)
     end
 
     def message
-      super + ' ' + _('for proxy') + ' ' + url
+      super + ' ' + proxy_exception_response(exception) +
```

Comment:
Nitpick, remove the extra space before `proxy_exception_response` as it'll cause a space before the colon and double-space otherwise.